### PR TITLE
Issue #4313: Split and Organize Checkstyle inputs by Test for checks in annotation package: AnnotationLocation

### PR DESCRIPTION
**Pull request by Kietzmann**
#4313 
While running receiving the same issue as here:
https://github.com/policeman-tools/forbidden-apis/issues/108
@MEZk 
How can I fix this in this case?
Thanks

### CODE REVIEW

**Comment by rnveach** [comment]
Did you try excluding the file responsible?

> InputAnnotationLocation4.class causes the exception. I excluded it and the exception disappear for now. 

https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L985

**Comment by Kietzmann** [comment]
@rnveach 
Done

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheckTest.java [hunk 1]
```diff
@@ -78,7 +80,7 @@ public void testIncorrect() throws Exception {
             "88: " + getCheckMessage(MSG_KEY_ANNOTATION_LOCATION, "MyAnnotation2", 10, 8),
             "98: " + getCheckMessage(MSG_KEY_ANNOTATION_LOCATION, "MyAnnotation2", 0, 3),
         };
-        verify(checkConfig, getPath("InputIncorrectAnnotationLocation.java"), expected);
+        verify(checkConfig, getPath("InpuAnnotationLocationtIncorrect.java"), expected);
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheckTest.java]: `Input` is missing tthe `t`.

**Comment by romani** [comment]
CI complains about damage of UTs expectations.

**Comment by Kietzmann** [comment]
@romani @rnveach 
Please review

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheckTest.java (hunk 1 above)
Comment (Kietzmann) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheckTest.java]: Fixed

**Pull request merged**